src/ContentBundle/Model/Content.php

```diff
@@ -278,6 +278,16 @@ class Content implements ContentInterface, EntityInterface, TemplatedInterface,
      */
     protected $publishAt;
 
+    /**
+     * @var int
+     *
+     * @JMS\Expose
+     * @ORM\Column(name="translation_group", type="integer", nullable=true)
```

Comment:
I'm not sure if this should be an integer... I expected the TranslationGroup to be a separate entity